website/docs/guides/installations.md

```diff
@@ -0,0 +1,70 @@
+---
+id: installation
+title: Manage Installations
+sidebar_position: 2
+---
+### Manage Installations
+
+### Before you start
+
+You can manage an installation assigned to a specific project and provider.
+Firstly you need to be registered to the accounting-system and then you need to contact the administrator of either the project or the provider, associated to the installation, in order to assign you one or more roles, on the installation.
```

Comment:
**Firstly,** you need to be registered to the accounting-system and then you need to contact the administrator of either the project or the provider, associated **with** the installation, ~~in order~~ to assign you one or more roles, on the installation.